modules/sdk-coin-avaxp/src/avaxp.ts

```diff
@@ -84,8 +84,7 @@ export class AvaxP extends BaseCoin {
   }
 
   verifyAddress(params: VerifyAddressOptions): boolean {
-    // TODO(STLX-16574): verifyTransaction
-    return true;
+    throw new MethodNotImplementedError();
```

Comment:
im little confused on why we throwing now? it will likely break?

Reply:
+1

Reply:
We don't use `verifyAddress` anywhere, and the comment said it was dependent on `verifyTransaction` - I don't think it is a good idea to keep the comment in there